### ISSUE TITLE
Can anyone help me set up Wikia connection?

### ISSUE THREAD

**Issue by SilvieSkydancer** [tosher/Mediawiker#50]
I'm brand new to Sublime Text. Like, BRAND new. I just got it tonight because I got sick of Wikia's template editor and its lack of word wrapping. So I'm here to ask how the heck I properly set up a Wikia connection! Trying to access a page with a general Wikia connection crashed the program, and after fiddling with the settings (as in, adding a specific mediawiki_site line for the wiki I wanted to access), I did get it to not crash...but instead of crashing, it instead brings up an error that says this:

"HTTP connection failed: httplib.HTTPResponse instance at 0x000000000403ADC8

I have no idea what this means (again, new to Sublime Text), but my API path is set as "api/v1/", since all the Wikia API info I could find led me to believe this was the right path. I think the API path might be the problem but I don't know how to set that up!

Anyone who's set this up with Wikia able to help me out here?  (And of course, tell me if I'm wrong with my API-path wild guessing, lol!)

**Comment by absay** [comment]
The API for each wiki is located at (wiki domain)/api.php, so the correct api path is simply "/", not "/api/v1/" (despite what the Wikia's API wiki tells you ;) ). Don't foget the username is case-sensitive.

Also, this is a huge coincidence but I posted a word warpping question on the Wikia's forum a few hours ago, and a response recommended me to use Notepad++ but since I prefer Sublime, I landed on this plugin.
